pkg/query-service/main.go

```diff
@@ -52,6 +52,8 @@ func main() {
 	flag.StringVar(&cacheConfigPath, "experimental.cache-config", "", "(cache config to use)")
 	flag.StringVar(&fluxInterval, "flux-interval", "5m", "(cache config to use)")
 	flag.StringVar(&cluster, "cluster", "cluster", "(cluster name - defaults to 'cluster')")
+	// Allow using the consistent naming with the signoz collector
+	flag.StringVar(&cluster, "cluster-name", "cluster", "(cluster name - defaults to 'cluster')")
```

Comment:
Wouldn't this override the `--cluster` always i.e even if someone provided `--cluster=my-cluster` the value would be default `cluster` from `--cluster-name` arg?

Reply:
@srikanthccv No, it won't be overridden unless the `--cluster-name` is provided since the flag parse was only invoked while the flag name was passed. That said, if users specify:

1. `--cluster ABC`, then the cluster will be ABC
2. `--cluster-name DEF`, then the cluster will be DEF
3. `--cluster ABC --cluster-name DEF`, then the cluster will be DEF
4. `--cluster-name DEF --cluster ABC`, then the cluster will be ABC

Reply:
@srikanthccv Any comment on this?

Reply:
I just verified. This looks good to me.

Reply:
@srikanthccv Thank you!